pkg/kapp/resources/mod_field_copy.go

```diff
@@ -120,6 +121,26 @@ func (t FieldCopyMod) apply(obj interface{}, path Path, fullPath Path, srcs map[
 				panic(fmt.Sprintf("Unknown array index: %#v", part.ArrayIndex))
 			}
 
+		case part.RegexObj != nil:
+			if part.RegexObj.Regex != nil {
```

Comment:
Since we are not doing anything when `RegexObj.Regex` is nil, maybe we can do something like `case part.RegexObj != nil && part.RegexObj.Regex != nil`?

Reply:
Adopted